bundles/org.dataflowanalysis.analysis.converter/src/org/dataflowanalysis/analysis/converter/microsecend/InformationFlow.java

```diff
@@ -0,0 +1,16 @@
+package org.dataflowanalysis.analysis.converter.microsecend;
+
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+public record InformationFlow(String sender,
```

Comment:
General comment: Every public method, every record and every class should have javadoc. Some are still missing